app/policies/account_policy.rb

```diff
@@ -14,6 +14,14 @@ def show?
     team.teammember?(@user.id)
   end
 
+  def update?
+    team.teammember?(@user.id)
+  end
+
+  def permitted_attributes
+    %i[accountname group_id description username password tag]
```

Comment:
I think password and username should never be updated directly !
cleartext_username, cleartext_password
see app/controllers/accounts_controller.rb
the username, password attrs of the model are the encrypted values ! 
very important to make sure this cannot be updated directly by a web request !